matchzoo/generators/list_generator.py

```diff
@@ -34,8 +34,12 @@ class ListGenerator(engine.BaseGenerator):
         ...                         columns=['id_left', 'id_right', 'label'])
         >>> left = pd.DataFrame(left, columns=['id_left', 'text_left'])
         >>> left.set_index('id_left', inplace=True)
+        >>> left['length_left'] = left.apply(lambda x: len(x['text_left']),
```

Comment:
This `lambda` thing appeared so many times that it is better to wrap it somehow.

Reply:
The best way is to define our own dataframe and add an additional function to infer the length. That would be a big step, we can discuss it~